ocfweb/docs/docs/services/vhost.md

```diff
@@ -135,9 +135,17 @@ does not process nor advise off-site hosting requests, which can be
 [submitted directly to the university][offsite] (however, feel free to keep
 us informed if you have an existing OCF account).
 
-In other words, to comply with university policy, the OCF may only provide
+   In other words, to comply with university policy, the OCF may only provide
```

Comment:
Hmm, I suppose so, but I think the bullet point thing is indented only because there is a bullet point, especially since the rest of the lines in the paragraph are not indented, so I see no reason for this paragraph to start with indentation either.